software/discourse.md

```diff
@@ -0,0 +1,22 @@
+# Discourse
+
+We use [Discourse](https://discourse.freistil.it/) for internal discussions that go beyond the context and/or scope of [Asana](/software/asana.html) and [Slack](/software/slack.html).
+
+Since Asana has it's limitations when it comes to quoting text and threaded replys (those simply don't exist there), for more complex discussions we have decided to use discourse, as it is specifically designed to handle long-term discussions with multiple partners and a potentially complex reply structure.
+
+Slack comes with it's own caveats - while it offers a single reply-thread per initial post there is no more complexity available beyond that, and additionally Slack does not lend itself well to conversations that are expected to be carried on over the current day, as slack is a more or less synchronous medium.  
+
+## Getting started
+
+Familiarize yourself with the [categories](https://discourse.freistil.it/categories) and the fine-grained options to "watch" any of those categories (see upper right (!) icon in any category).
+
+Watch any categories you like/will likely need for your work.  
+
+## How to have a good discourse discussion
+
+* To clearly separate concerns, create a new topic rather than highjacking an existing one.
+* Put your topic in the appropriate category and apply helpful tags.
+* Take enough time to formulate your topic/reply, as it is more important for your text to be understandable and complete than to be available quickly.
+* Refrain from using humor/sarcasm/trolling in discourse discussions as this can quickly derail the conversation in progress.
```

Comment:
Add "and can cause constant doubt in your readers if you're serious or not."
Lose "in progress"